### DISCUSSION TITLE
Welcome to raycast-g4f Discussions!

### DISCUSSION THREAD

**Discussion by XInTheDark** [XInTheDark/raycast-g4f#70] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by binaryblitz10** [comment]
Hey,

Great project!
However, I was just wondering how you are able to provide this for free without using our own API Key?

**Comment by XInTheDark** [comment]
@binaryblitz10 hey! The extension basically works by either using free APIs, or reverse engineering websites that provide such free services. For more details, you can check out the https://github.com/xtekky/gpt4free project :)
